composer/loggers/in_memory_logger.py

```diff
@@ -8,17 +13,43 @@
 from composer.core.logging.logger import TLogDataValue
 from composer.core.time import Timestamp
 
+__all__ = ["InMemoryLogger"]
+
 
 class InMemoryLogger(LoggerCallback):
-    """Stores all logging calls in memory.
+    """Logs metrics to dictionary objects that persist in memory throughout training. Useful for collecting and plotting
+    data inside notebooks.
+
+    Example usage:
+        .. testcode::
+
+            from composer.loggers import InMemoryLogger
+            from composer.trainer import Trainer
+            from composer.core.logging import LogLevel
+            logger = InMemoryLogger(
+                log_level=LogLevel.BATCH
+            )
+            trainer = Trainer(
+                model=model,
+                train_dataloader=train_dataloader,
+                eval_dataloader=eval_dataloader,
+                max_duration="1ep",
+                optimizers=[optimizer],
+                loggers=[logger]
+            )
```

Comment:
can you add a line here showing actually how to access the data in the dictionary?

Reply:
Example added. It's not the cleanest looking thing. I'm excited for the logger revamp.

Reply:
Was something new pushed? I don't see anything after the `trainer=Trainer(..)` line.